cmds/ip/ip.go

```diff
@@ -66,25 +69,86 @@ func dev() netlink.Link {
 	return iface
 }
 
-func showips() {
+func showLinks(withAddresses bool) {
```

Comment:
would it be make sense to have this and the other ones take an io.Writer for fmt to use in an Fprintf?

Reply:
I didn't look with attention implementation of the other ones prior to start this work... But I agree. It would be better at least for code style consistency to use the `io.Writer` here also. I'll update this one.